crates/iota-grpc-api/tests/checkpoint_e2e.rs

```diff
@@ -205,7 +207,11 @@ async fn test_stream_full_checkpoint_data() {
                 panic!("Expected data, got summary")
             }
         };
-        assert_eq!(checkpoint_data.checkpoint_summary.sequence_number, 2);
+        if let Some(v1_data) = checkpoint_data.as_v1() {
```

Comment:
Also maybe `match` here instead?

Reply:
Thanks for pointing this out! Fixed it at [0469376](https://github.com/iotaledger/iota/pull/7511/commits/0469376723d56074ca9fa703d333e4a9cd104fc8).